package.json

```diff
@@ -53,7 +53,7 @@
 		"wd": "^1.11.1"
 	},
 	"scripts": {
-		"postinstall": "patch-package && npm ci --prefix gutenberg && cd jetpack/projects/plugins/jetpack && npx pnpm install",
+		"postinstall": "patch-package && npm run i18n:update:skip-if-cache && npm ci --prefix gutenberg && cd jetpack/projects/plugins/jetpack && npx pnpm install",
```

Comment:
Translations are required for running the editor, and since they are not versioned, it's important that we guarantee their presence when installing the dependencies. This behavior follows the same approach we do on Gutenberg ([reference](https://github.com/WordPress/gutenberg/blob/fe9b08a2fd5dacb5948c3c5fc1af875f84e2142e/packages/react-native-editor/package.json#L95)).